website/docs/user-guides/manage-app-dashboard.md

```diff
@@ -4,4 +4,86 @@ title: Creating and Updating an Application Dashboard
+___
+
+Observing chaos injections and deriving insights from the chaos events is the core part of Chaos Observability. Whenever a chaos workflow run is performed its impact may be observed across different services and applications. Application dashboards provide a systematic visualization of all such events along with the performance of the individual service/application. To learn more about the dashboard schema architecture [click here](https://raw.githubusercontent.com/litmuschaos/litmus/master/monitoring/portal-dashboards/schema.json)
+
+### Prerequisite 
+To configure an application dashboard, you must first add a data source. To learn more about data sources [click here](configure-datasource)
+
+  - If no data source is available, a banner on the application dashboard tab requests you to add it before configuring a dashboard. 
+
+<figure>
+<img src={require('../assets/user-guides/observability/setup/manage-application-dashboard-no-data-source-dashboard.png').default} />
+<i>Application dashboard with no configured data source</i>
+</figure>
+
+  - If a data source becomes inactive after dashboard creation then you will be requested to either reconfigure the dashboard with another data source or update the existing data source.
+
+
+
+### Table for configured dashboards
+
+  All the configured dashboards are listed under the Dashboards table. The table provides you all the details related to the dashboard like the name of the dashboard, the agent with which it has been connected, type of the dashboard, connected data source type and the last date it was opened.
+
+  For all the dashboards listed in the table, you can filter the dashboard with respect to agent name, dashboard-type and time period for dashboard creation.
+
+<figure>
+<img src={require('../assets/user-guides/observability/setup/manage-application-dashboard-table.png').default} />
+<i>Application dashboard table with a configured dashboard</i>
+</figure>
+
+On clicking on more options, you can select `View` to open the dashboards, click on configure to make changes in the existing dashboard, you can also download the dashboard in `JSON` format and click on `Delete` to remove the dashboard.
+
+
+
+### Create dashboard
+
+- To add a dashboard, click on the `Create dashboard` button placed at the right corner.
+
+<figure>
+<img src={require('../assets/user-guides/observability/setup/manage-application-dashboard-create-dashboard.png').default} />
+<i>Creating a dashboard</i>
+</figure>
+
+- **Choose a dashboard-type:**
+<figure>
+<img src={require('../assets/user-guides/observability/setup/manage-application-dashboard-dashboard-type.png').default} />
+<i>Select dashboard type</i>
+</figure>
+
+You need to select the dashboard type from the following options:
+
+  - *Predefined dashboards:* Litmus provides a set of predefined dashboards for different applications and use-cases like Node metric, Pod metric, Sock Shop etc along with the chaos events and chaos exporter metrics.
```

Comment:
```suggestion
  - *Predefined dashboards:* ChaosCenter provides a set of predefined dashboards for different applications and use-cases like Node metric, Pod metric, Sock Shop etc along with the chaos events and chaos exporter metrics.
```

Reply:
For this use case we will be using Litmus